docs/source/clouds/cloud_training.rst

```diff
@@ -4,39 +4,45 @@
 Cloud Training
 ##############
 
-Lightning has a native solution for training on AWS/GCP at scale.
-Go to `grid.ai <https://www.grid.ai/>`_ to create an account.
+Lightning makes it easy to scale your training, without the boilerplate.
+If you want to train your models on the cloud, without dealing with engineering infrastructure and servers, you can try `Grid.ai <https://www.grid.ai/>`_.
```

Comment:
```suggestion
If you want to train your models in the cloud, without dealing with engineering infrastructure and servers, you can try `Grid.ai <https://www.grid.ai/>`_.
```
Is it "in" or "on"? Usually when we say "we train a model on X" we mean a task or a dataset.